medusa/tv/series.py

```diff
@@ -2188,11 +2192,13 @@ def want_episode(self, season, episode, quality, forced_search=False,
         )
         return should_replace
 
-    def get_overview(self, ep_status, backlog_mode=False, manually_searched=False):
+    def get_overview(self, ep_status, ep_quality, backlog_mode=False, manually_searched=False):
         """Get the Overview status from the Episode status.
 
         :param ep_status: an Episode status
         :type ep_status: int
+        :param ep_quality: an Episode status
```

Comment:
*quality

Also: A few lines below we do:
`ep_status = try_int(ep_status) or UNSET`
Should there be one for quality too?
`ep_quality = try_int(ep_quality) or Quality.NA`

Reply:
I've removed the try_int and also the or. If we pass something that isn't valid we need to find the cause of it, try_int is generally a mistake.